python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -1098,7 +1180,10 @@ def _fast_slow_function_call(
             _slow_function_call()
             with disable_module_accelerator():
                 result = func(*slow_args, **slow_kwargs)
-    return _maybe_wrap_result(result, func, *args, **kwargs), fast
+    result = _maybe_wrap_result(result, func, *args, **kwargs)
+    if block_transfer_to_fast and isinstance(result, _FastSlowProxy):
+        result.force_state(_State.SLOW)
```

Comment:
So ultimately all transfer blocking is handled at the level of forcing a specific state, which in turn blocks transfers out of that state?